src/commands/function-stats.json

```diff
@@ -18,6 +18,64 @@
             "NONDETERMINISTIC_OUTPUT",
             "REQUEST_POLICY:ALL_SHARDS",
             "RESPONSE_POLICY:SPECIAL"
-        ]
+        ],
+        "reply_schema": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "running_script": {
+                    "description": "information about the running script.",
+                    "oneOf": [
+                        {
+                            "description": "If there's no in-flight function",
+                            "type": "null"
+                        },
+                        {
+                            "description": "a map with the information about the running script",
+                            "type": "object",
+                            "additionalProperties": false,
+                            "properties": {
+                                "name": {
+                                    "description": "the name of the function.",
+                                    "type": "string"
+                                },
+                                "command": {
+                                    "description": "the command and arguments used for invoking the function.",
+                                    "type": "array",
+                                    "items": {
+                                        "type": "string"
+                                    }
+                                },
+                                "duration_ms": {
+                                    "description": "the function's runtime duration in milliseconds.",
+                                    "type": "integer"
+                                }
+                            }
+                        }
+                    ]
+                },
+                "engines": {
+                    "description": "A map when each entry in the map represent a single engine.",
+                    "type": "object",
+                    "patternProperties": {
+                        "^.*$": {
+                            "description": "Engine map contains statistics about the engine",
+                            "type": "object",
+                            "additionalProperties": false,
```

Comment:
i think this one should be dropped. i don't wanna restrict which fields are in this map